package.json

```diff
@@ -3,22 +3,40 @@
   "version": "1.5.0",
   "workspaces": [
     "packages/daemon",
-    "packages/wallet-service"
+    "packages/wallet-service",
+    "packages/common"
```

Comment:
suggestion: Order this alphabetically, as it will probably be automatically fixed at some point in the future
```js,
    "packages/common",
    "packages/daemon",
    "packages/wallet-service"
```

Reply:
Done in 661c1478522b0b101dbb8f08015e2b466d2f3d2c